<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 232, 232);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Styczeń</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>Luty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Marzec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Kwiecień</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>Maj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Czerwiec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Lipiec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Sierpień</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Wrzesień</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>Październik</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Listopad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Grudzień</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Wszystkie miesiące</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Rozpoczęci klienci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(73, 84, 100);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Cofnij</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(73, 84, 100);</string>
   </property>
   <property name="text">
    <string>Rok: Aktualny</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>LogoSample_ByTailorBrands.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
